backend/src/junior/dto/season-expired.dto.ts

```diff
@@ -0,0 +1,8 @@
+import { IsNotEmpty } from 'class-validator';
+
+export class SeasonExpiredDto {
+
+    @IsNotEmpty()
+    readonly expireDate: string;
```

Comment:
Do you know if you can easily add validation rules for these fields? We'll want to pass the date to the backend as an ISO string, for example `"2021-06-17T00:00:00.000Z"`, and the endpoint should return `400 bad request` on a validation error.

Reply:
It's possible and quite simple:
```
import { IsNotEmpty, IsDateString } from 'class-validator';

export class SeasonExpiredDto {
  @IsNotEmpty()
  @IsDateString()
  readonly expireDate: string;
}
```